src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
```

Comment:
Unnecessary new line - you didn't put it here in other classes. Keep your code consistent.